docs/assistant/getting-started/account-types.md

```diff
@@ -0,0 +1,13 @@
+---
+id: account-types
+title: Account Types
+sidebar_label: Account Types
+---
+
+There are three Assistant Pro account types: Free, Personal and Team. 
+ ​​
+* The Assistant WordPress Plugin is free to all users  
+* The Assistant Pro Cloud App has both free and premium options
+* There are two premium options, one for individuals and one for teams
```

Comment:
{ The main difference in premium account types is that one is for individuals and one is for teams.  Maybe this could be made more clear with a second level of bullets. }

Updated Copy:
* The Assistant WordPress Plugin is free to all users  
* The Assistant Pro Cloud App has both free and premium options
* There are two premium options
            -Premium Accounts for Individual Users
            -Premium Accounts for Team Users